index.html

```diff
@@ -286,48 +290,50 @@ <h2>Languages</h2>
 							</ul>							
 						</li>
```

Comment:
- [ ] The `Get my resume` button in your **About me section**  now looks a bit off-design plus it doesn't have the hover effect as requested by the previous reviewer 👨🏽‍🏫 Kindly fix this by adding the hover effects and resizing the button 🎯.

![image](https://user-images.githubusercontent.com/43172164/200057807-f7eec3aa-960c-4415-ba22-5a9564d6671f.png)
![image](https://user-images.githubusercontent.com/43172164/200058292-22e235ec-2b3c-48be-b9ef-1b3ae268e149.png)

Reply:
The button also appears fine on my end. Kindly advise.